test/test_linalg.py

```diff
@@ -1475,6 +1475,7 @@ def run_test_case(input, p):
                 actual = torch.linalg.cond(input, p)
                 self.assertEqual(actual, expected)
 
+    @skipIfRocm
```

Comment:
Is there an issue tracking this regression?

Reply:
https://github.com/pytorch/pytorch/issues/55552